docssrc/src/velocity_intro.md

```diff
@@ -0,0 +1,115 @@
+# Velocity
+
+> **Developer's Note:**
+>
+> The CommandAPI hasn't been released for Velocity yet.
+> We do, however, offer snapshot builds. This small section on Velocity will outline how to get the snapshot builds and what limitations the CommandAPI currently has on Velocity.
+
+## Adding the snapshot repository with Maven or Gradle
+
+Because we do not have an official release yet, the snapshot builds are not published in the Maven Central repository. Instead you need to add our snapshot repository:
+
+<div class="multi-pre">
+
+```xml,pom.xml
+<repositories>
+    <repository>
+        <id>oss.sonatype.org-snapshot</id>
+        <url>https://s01.oss.sonatype.org/content/repositories/snapshots</url>
+    </repository>
+</repositories>
+```
+
+```gradle,build.gradle
+repositories {
+    maven {
+        url = "https://s01.oss.sonatype.org/content/repositories/snapshots"
+    }
+}
+```
+
+```gradle,build.gradle.kts
+repositories {
+    maven {
+        url = uri("https://s01.oss.sonatype.org/content/repositories/snapshots")
+    }
+}
+```
+
+</div>
+
+## Adding the dependency
+
+As mentioned, Velocity can only be accessed with snapshot builds. These snapshot build version are following standard semantic versioning and thus have the `-SNAPSHOT` suffix:
+
+<div class="multi-pre">
+
+```xml,pom.xml
+<dependencies>
+    <dependency>
+        <groupId>dev.jorel</groupId>
+        <artifactId>commandapi-velocity-shade</artifactId>
+        <version>9.4.0-SNAPSHOT</version>
+    </dependency>
+</dependencies>
+```
+
+```gradle,build.gradle
+dependencies {
+    compile "dev.jorel:commandapi-velocity-shade:9.4.0-SNAPSHOT"
+}
+```
+
+```gradle,build.gradle.kts
+dependencies {
+    compileOnly("dev.jorel:commandapi-velocity-shade:9.4.0-SNAPSHOT")
+}
+```
```

Comment:
Should we add these to the `update.sh` script?

https://github.com/JorelAli/CommandAPI/blob/800372c52ce7b3ca4195224fc1487d31d5cb48d7/update.sh#L11-L22

Maybe not right now. The versions on this documentation page aren't supposed to be exactly up-to-date. They're supposed to be the snapshot version before the newest version.